pkg/madmin/info-commands.go

```diff
@@ -288,3 +288,45 @@ func (adm *AdminClient) ServerMemUsageInfo() ([]ServerMemUsageInfo, error) {
 
 	return info, nil
 }
+
+// NetPerfInfo network performance information.
+type NetPerfInfo struct {
+	Addr     string        `json:"addr"`
+	ReadPerf time.Duration `json:"readPerf"`
+	Error    string        `json:"error,omitempty"`
+}
+
+// NetPerfInfo - Returns network performance information of all cluster nodes.
+func (adm *AdminClient) NetPerfInfo() (map[string][]NetPerfInfo, error) {
+	v := url.Values{}
+	v.Set("perfType", "net")
```

Comment:
```suggest
v.Set("type", "net")
```

Reply:
Can you tell me why?

Reply:
the query value `type` is predicated on the resource `performance` - it is redundant to name query as perfType here. 
`GET /v1/performance?type=net` seems more idiomatic - just a suggestion here, feel free to ignore.

Reply:
Note that I am adding to existing API and I don't know why names were chosen that way. This is out of scope of this PR.